js/bybit.js

```diff
@@ -3503,11 +3398,14 @@ module.exports = class bybit extends Exchange {
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
         await this.loadMarkets ();
+        this.checkRequiredSymbol ('createOrder', symbol);
         const market = this.market (symbol);
         symbol = market['symbol'];
-        const enableUnifiedMargin = await this.isUnifiedMarginEnabled ();
+        const [ enableUnifiedMargin, enableUnifiedAccount ] = await this.isUnifiedEnabled ();
         const isUSDCSettled = market['settle'] === 'USDC';
-        if (market['spot']) {
+        if (enableUnifiedAccount) {
```

Comment:
I think this check should also include linear/inverse contracts.

My account is not unified - but based on the [create order](https://bybit-exchange.github.io/docs/v5/order/create-order) documentation, it should work for contracts, too - also on non-unified accounts.

Reply:
@xmatthias it looks like only support linear for UTA account, and linear / inverse for the normal account.